dashboard/src/components/proposals/ProposalBlock.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div v-if="proposal && proposalLikes != null" class="flex justify-between">
+    <div class="flex flex-col gap-3 justify-between">
+      <h3 class="text-xl font-semibold">
+        <a
+          :href="redirectRouteToSameWindow(proposal.route)"
+          class="hover:text-green-500 transition-colors duration-300 cursor-pointer"
+        >
+          {{ proposal.talk_title }}
+        </a>
+      </h3>
+      <div
+        class="flex flex-col gap-3 items-start mt-1 md:flex-row md:items-center"
+      >
+        <div class="flex items-center text-xs font-semibold">
+          <span class="bg-gray-200 px-2 py-1 text-gray-600 rounded-sm mr-2">{{
+            proposal.session_type.toUpperCase()
+          }}</span>
+          <div>
+            <span
+              class="px-2 py-1 rounded-sm md:hidden"
+              :class="getStatusClass(proposal.status)"
+              >{{ proposal.status.toUpperCase() }}</span
+            >
+          </div>
+        </div>
+        <div v-if="proposal.status == `Approved`" class="flex text-sm">
+          <SpeakerIcon />
+          <span>
+            {{ proposal.full_name }}
+          </span>
+        </div>
+      </div>
+    </div>
+    <div class="flex flex-col gap-3 items-end">
+      <div class="flex items-center gap-1 bg-gray-200 px-1.5 py-1 rounded-sm">
+        <LikesIcon />
+        <span class="text-xs font-semibold">{{ proposalLikes }}</span>
+      </div>
+      <div class="mt-2">
+        <span
+          class="px-2 py-1 rounded-sm text-xs font-semibold hidden md:block"
+          :class="getStatusClass(proposal.status)"
+          >{{ proposal.status.toUpperCase() }}</span
+        >
```

Comment:
Am i seeing this twice? please confirm this was not a mistake

Reply:
It's not a mistake - https://github.com/fossunited/fossunited/pull/589/files/cb99fca591d632446d2fc4ebcbeb73a2b0cc31cf#r1742301231